c2corg_api/scripts/jobs/scheduler.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import sys
+import os
+import signal
+
+from pyramid.scripts.common import parse_vars
+from pyramid.paster import get_appsettings, setup_logging
+
+from sqlalchemy import engine_from_config
+
+from c2corg_api.models import Base, DBSession
+from c2corg_api.jobs import configure_scheduler_from_config
+
+log = logging.getLogger('c2corg_api_background_jobs')
```

Comment:
Yes I tested and it worked. Probably because I am logging with level > `info`.
I will add the same kind of config as yours anyway.